### PR TITLE
Update dependencies and peer dependencies

### PR DESCRIPTION
**Pull request by aalmazan**
## Description of the change

Update peer dependencies and other dependencies across all packages.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

Fixes <add Issue here, can be in format #xx>

## Dev Checklist

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development
- [x] No previous tests unrelated to the changed code fail in development

### CODE REVIEW

File: packages/react-signature-frame/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
+## [@anvilco/react-signature-frame@1.7.0](https://github.com/anvilco/react-ui/compare/@anvilco/react-signature-frame@1.6.0...@anvilco/react-signature-frame@1.7.0) - 2022-03-31
```
Comment (benogle) [inline packages/react-signature-frame/CHANGELOG.md:5]: Does the changelog get updated when publishing? Like after this PR is merged?

File: packages/react-signature-modal/package.json [hunk 2]
```diff
@@ -32,15 +32,32 @@
...
+    "prop-types": "^15.8.1",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "react-modal": "^3.11.2"
+    "react-modal": "^3.14.4",
+    "webpack": "^5.70.0",
+    "webpack-cli": "^4.9.2"
   },
   "peerDependencies": {
     "prop-types": "^15.6.0",
```
Comment (benogle) [inline packages/react-signature-modal/package.json:58]: No new majors here or in react-modal, eh?

**Review by benogle — APPROVED** [review]

File: packages/react-signature-modal/package.json (hunk 2 above)
Comment (aalmazan) [reply packages/react-signature-modal/package.json:58]: Yeah no new majors there. I used `yarn upgrade-interactive --latest` and chose the ones that don't have such a big impact and that's what we got.

File: packages/react-signature-frame/CHANGELOG.md (hunk 1 above)
Comment (aalmazan) [reply packages/react-signature-frame/CHANGELOG.md:5]: Changelog updates are part of `yarn build:changelog`

**Pull request merged**